var/spack/repos/builtin/packages/gpgme/package.py

```diff
@@ -33,7 +33,7 @@ class Gpgme(AutotoolsPackage):
 
     depends_on("gnupg", type="build")
     depends_on("libgpg-error", type="build")
-    depends_on("libassuan", type="build")
+    depends_on("libassuan@:2.5.7", type="build")
```

Comment:
While we're at it, there's a min version too (this one since 1.10.0 so for all versions), and there may be more bugfix releases in v2:
```suggestion
    depends_on("libassuan@2.4.2:2", type="build")
```

Reply:
Curious: Is this really only a build dependency and not a link dependency?